src/compiler/python_generator.cc

```diff
@@ -156,6 +156,19 @@ grpc::string ModuleName(const grpc::string& filename) {
   return basename + "_pb2";
 }
 
+// TODO(protobuf team): Export `ModuleAlias` from protobuf's
```

Comment:
Is this [protobuf issue 888](https://github.com/google/protobuf/issues/888)? If so, write it as `TODO(https://github.com/google/protobuf/issues/888): Export...`.